setup.py

```diff
@@ -1,60 +0,0 @@
-#!/usr/bin/env python
```

Comment:
@septimit It's a GitHub release and we do not require this file. Is that the reason for deleting this file?

Reply:
I propose we only use the official tests from dbt-utils and thus removing the custom test suite. This file does not need to be here anymore.